src/app/services/question.service.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { catchError, map, Observable, throwError } from 'rxjs';
+import { QuestionModel } from './model/question.model';
+import { API_ENDPOINTS } from '../utils/constants';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class QuestionService {
+  constructor(private http: HttpClient) {}
+  
+  get(category: number): Observable<QuestionModel[]> {
```

Comment:
is this a method? let's rename it to getQuestions() or something, to prevent confusing with getters which we define via get.